hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/spark/sql/hudi/streaming/HoodieStreamSourceV2.scala

```diff
@@ -45,26 +41,15 @@ import org.apache.spark.sql.types.StructType
   * @param parameters
   */
 // TODO(yihua): handle V1/V2 checkpoint
```

Comment:
oh can we also cleanup this TODO please.

Reply:
Yeah. Will do a separate PR for this.